test/speed.c

```diff
@@ -37,8 +37,9 @@
 static unsigned long long cpucycles(void)
 {
   unsigned long long result;
-  __asm volatile(".byte 15;.byte 49;shlq $32,%%rdx;orq %%rdx,%%rax"
-    : "=a" (result) ::  "%rdx");
+//   __asm volatile(".byte 15;.byte 49;shlq $32,%%rdx;orq %%rdx,%%rax"
+//     : "=a" (result) ::  "%rdx");
+  result = 0;
```

Comment:
Why do you have that change?

Reply:
This is assembly to read cycle count for x86 platform. On current MBP M1, I use wall clock as benchmark timing, this approach extend the benchmark code to other platforms, not only x86. 
Because right now writing the cycle count for M1 takes more time.